cliconfig/file.go

```diff
@@ -120,19 +120,16 @@ func parseLine(line string) (key string, value string, err error) {
 	}
 
 	if len(splitString) != 2 {
-		err = errors.New("Can't separate key from value")
-		return
+		return "", "", fmt.Errorf("can't separate key from value in string %q, no valid separators (= or :) found", line)
 	}
 
 	// Parse the key
-	key = splitString[0]
-	if strings.HasPrefix(key, "export") {
-		key = strings.TrimPrefix(key, "export")
-	}
+	key := splitString[0]
+	key = strings.TrimPrefix(key, "export")
 	key = strings.Trim(key, " ")
```

Comment:
[Optional] Should we trim other kinds of spaces? `strings.TrimSpace`?

Reply:
yeah, i think that's probably a good call. some people might be using (shudder) tabs for indentation.